lib/css/modules/_userTagger.scss

```diff
@@ -3,6 +3,7 @@
 	position: absolute;
 	width: 336px;
 	box-sizing: border-box;
+	z-index: 35;
```

Comment:
Think all of em are more than high enough to get job done :p Have just gone with ` $zindex-res-hover`